### PR TITLE
Button component

### PR DESCRIPTION
**Pull request by desplodur**
Please take a look at my button component.
Especially on the styled components variants. 🤖 🚀

### CODE REVIEW

**Review by PhilippDemmelmair — APPROVED** [review]
Looks good

File: src/components/Form/index.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
+import Button from '../Button';
+
 import StyledForm from './styled';
 
 export default function Form({onSubmit, placeholderText, submitButtonText}) {
 	return (
 		<StyledForm onSubmit={onSubmit}>
 			<input type="text" name="inputField" placeholder={placeholderText} required />
-			<button type="submit" value="Submit the form">
+			<Button submitButton={true} type="submit" value="Submit the form">
```
Comment (stefankorder) [inline src/components/Form/index.js:9]: Why do you double the submit information? If you declare your button as type submit you don't need to set an attribute called submit button as true.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! Just one suggestion!

File: src/components/Form/index.js (hunk 1 above)
Comment (desplodur) [reply src/components/Form/index.js:9]: Thanks for the Information! :)

**Pull request merged**